admin/includes/languages/english/whos_online.php

```diff
@@ -40,7 +40,8 @@
 define('TEXT_WHOS_ONLINE_FILTER_ADMINS', 'Exclude Admin IP Addresses?');
 
 define('WHOIS_TIMER_REMOVE', 1200); // seconds when removed from whos_online table - 1200 default = 20 minutes
-define('WHOIS_TIMER_INACTIVE', 180); // seconds when considered inactive - 180 default = 3 minutes
+// moved to english.php 
+// define('WHOIS_TIMER_INACTIVE', 180); // seconds when considered inactive - 180 default = 3 minutes

```

Comment:
Not loving this change. Breaks up the context.

Sure, to use the same constant elsewhere it must be shared, but that makes me wonder how important this even is ...

Reply:
Could move all 3 to english.php Without this, the 2 pages are out of sync - wouldn't you call that a confusing bug?  One of my customers complained as soon as I put 1.5.6 on their site.

Reply:
> Could move all 3 to english.php

Sigh. Yes we certainly could. 

But then everyone's gonna complain that these are not language-related at all, and complain that we're cluttering english.php, and want more configuration switches that are already too cumbersome ...

Reply:
extra_configures?  Put in db?  What would you like to do, Chris?